sklearn/utils/seq_dataset.pxd.tp

```diff
@@ -1,13 +1,14 @@
-"""Dataset abstractions for sequential data access. """
-
-cimport numpy as np
+{{py:
```

Comment:
I put it just below actually. First the code defining the template (in the braces {{py blablabla}}) and then the template itself. Only the second part is generated.